### PR TITLE
Changed methods, and functionality

### PR DESCRIPTION
**Pull request by puzzledShark**
I changed browse to init
init now intializes the iframe as an empty iframe, and thats all it does, no longer takes in url
update is now available as a public API and it takes in a url, and replaces the src of the iframe we create with it

### CODE REVIEW

File: browser/iframe-browser.js [hunk 1]
```diff
@@ -60,23 +70,15 @@ define(function (require, exports, module) {
...
-        var iframeConfig = {
-            id: "bramble-iframe-browser",
-            frameborder: 0
-        };
-
+    function update(url) {
         // If url has been sent, make the iFrames' src that of the url
         if(url) {
-            iframeConfig.src = url;
+            //iframeConfig.src = url;
```
Comment (sedge) [inline browser/iframe-browser.js]: Remove this completely rather than comment.

**Comment by sedge** [comment]
Looks good to me.

File: browser/iframe-browser.js [hunk 2]
```diff
@@ -10,21 +10,31 @@ define(function (require, exports, module) {
...
+
     /*
-     * Publicly avaialble function used to create an iframe within the second-panel
-     * url: Takes one argument of a url to a blob for the _update() function to use 
-     *      - if undefined we will not fill the iframe with an src
+     * Publicly avaialble function used to create an empty iframe within the second-panel
      */
-    function browse(url) {
+    function init() {
         //Get current GUI layout
```
Comment (humphd) [inline browser/iframe-browser.js:22]: You probably only want this called once, so you might want to add this to the start of this function:

``` js
if(_panel) {
  return;
}
```

Comment (puzzledShark) [inline browser/iframe-browser.js:22]: would I need to make _panel global then?

File: browser/iframe-browser.js [hunk 3]
```diff
@@ -60,23 +70,14 @@ define(function (require, exports, module) {
...
-            frameborder: 0
-        };
-
+    function update(url) {
         // If url has been sent, make the iFrames' src that of the url
         if(url) {
-            iframeConfig.src = url;
+            var $iframe = $("#bramble-iframe-browser");
+            if ( $iframe.length ) {
+                $iframe.attr('src',url);
```
Comment (humphd) [inline browser/iframe-browser.js]: Watch your spaces around arguments.  You want to have the parens be tight to the args, but have a space after commas.

File: browser/iframe-browser.js [hunk 4]
```diff
@@ -60,23 +74,17 @@ define(function (require, exports, module) {
...
-        var iframeConfig = {
-            id: "bramble-iframe-browser",
-            frameborder: 0
-        };
-
+    function update(url) {
         // If url has been sent, make the iFrames' src that of the url
         if(url) {
-            iframeConfig.src = url;
+            //Get the iframe
```
Comment (humphd) [inline browser/iframe-browser.js]: Remove these comments that don't add anything (i.e., it's obvious that it's getting the iframe, checking if it exists, and setting the url).

**Comment by humphd** [comment]
One nit, @sedge can you do a final r? too please?

**Comment by sedge** [comment]
R+

**Pull request merged**